utils/website_crawler.py

```diff
@@ -0,0 +1,54 @@
+from HTMLParser import HTMLParser
+import urllib
+
+
+class WebsiteCrawler(HTMLParser):
+    def __init__(self, raw_html, website_url):
+        HTMLParser.__init__(self)
+        print("Crawling possible websites...")
+        self.website_set = set()
+        self.website_url = website_url
+        self.raw_html = raw_html
+
+        self.website_set.add(website_url)
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "a":
+            for name, value in attrs:
+                if name == "href":
+                    if self.website_url not in value:
```

Comment:
You're right, we should create a method which handles if the `website_url` is in the base URL (before the first slash), not in the whole String.
The method would be something like this:
```
def get_base_url(self, raw_url):
      # check if '/' is present first, if it's not, return raw_url
      split_url = raw_url.split("/", 1)
      return split_url[0]
```

---- 

The condition would now be 
`                      if self.website_url not in self.get_base_url(value):`